cmd/constants/markets.go

```diff
@@ -0,0 +1,5340 @@
+			  {
+				"name": "kucoin_ws",
+				"off_chain_ticker": "WOO-USDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "okx_ws",
+				"off_chain_ticker": "WOO-USDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "mexc_ws",
+				"off_chain_ticker": "WOOUSDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  }
+			]
+		  },
+		  "XLM/USD": {
+			"ticker": {
+			  "currency_pair": {
+				"Base": "XLM",
+				"Quote": "USD"
+			  },
+			  "decimals": 10,
+			  "min_provider_count": 3,
+			  "enabled": true
+			},
+			"provider_configs": [
+			  {
+				"name": "binance_api",
+				"off_chain_ticker": "XLMUSDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "bybit_ws",
+				"off_chain_ticker": "XLMUSDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "coinbase_ws",
+				"off_chain_ticker": "XLM-USD"
+			  },
+			  {
+				"name": "kraken_api",
+				"off_chain_ticker": "XXLMZUSD"
+			  },
+			  {
+				"name": "kucoin_ws",
+				"off_chain_ticker": "XLM-USDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "mexc_ws",
+				"off_chain_ticker": "XLMUSDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "okx_ws",
+				"off_chain_ticker": "XLM-USDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "crypto_dot_com_ws",
+				"off_chain_ticker": "XLM_USDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "bitstamp_ws",
+				"off_chain_ticker": "xlmusd"
+			  }
+			]
+		  },
+		  "XRP/USD": {
+			"ticker": {
+			  "currency_pair": {
+				"Base": "XRP",
+				"Quote": "USD"
+			  },
+			  "decimals": 10,
+			  "min_provider_count": 3,
+			  "enabled": true
+			},
+			"provider_configs": [
+			  {
+				"name": "binance_api",
+				"off_chain_ticker": "XRPUSDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "bybit_ws",
+				"off_chain_ticker": "XRPUSDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "coinbase_ws",
+				"off_chain_ticker": "XRP-USD"
+			  },
+			  {
+				"name": "gate_ws",
+				"off_chain_ticker": "XRP_USDT",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "huobi_ws",
+				"off_chain_ticker": "xrpusdt",
+				"normalize_by_pair": {
+				  "Base": "USDT",
+				  "Quote": "USD"
+				}
+			  },
+			  {
+				"name": "kraken_api",
+				"off_chain_ticker": "XXRPZUSD"
+			  },
+			  {
+				"name": "kucoin_ws",
+				"off_chain_ticker": "XRP-USDT",
```

Comment:
Can we have these as native go structs rather than as json that we / unmarshal?

Reply:
replicating the core market is gonna take me a fat minute. this effectively accomplishes the same thing